src/stdio.jl

```diff
@@ -52,6 +84,43 @@ function watch_stream(rd::IO, name::AbstractString)
     end
 end
 
+function label_timed_task(name)
+    if verbose::Bool && !haskey(task_local_storage(), :IJulia_task)
+        label = "send $name task"
+        task_local_storage(:IJulia_task, label)
+        @vprintln("$label labelled")
+    end
+end
+
+function send_stdout(t::Timer)
+    name = "stdout"
+    label_timed_task(name)
+    send_stream(name)
+end
+
+function send_stderr(t::Timer)
```

Comment:
It seems like `send_stdout`, `send_stderr`, and `label_timed_task` can be combined into a single function:
```jl
function send_stdio(t::Timer, name)
    if verbose::Bool && !haskey(task_local_storage(), :IJulia_task)
        task_local_storage(:IJulia_task, "send $name task")
    end
    send_stream(name)
end
```